Sources/MaterialUI/Toggle.swift

```diff
@@ -9,7 +9,9 @@
 import SwiftUI
 
 public struct MaterialSwitchToggleStyle: ToggleStyle {
-	// https://github.com/flutter/flutter/blob/ee032f67c734e607d8ea5c870ba744daf4bf56e7/packages/flutter/lib/src/material/switch.dart#L19
+	@Environment(\.materialAccent) var accentColor: Color
+    
+    // https://github.com/flutter/flutter/blob/ee032f67c734e607d8ea5c870ba744daf4bf56e7/packages/flutter/lib/src/material/switch.dart#L19
```

Comment:
spaces -> tabs again
```suggestion
	// https://github.com/flutter/flutter/blob/ee032f67c734e607d8ea5c870ba744daf4bf56e7/packages/flutter/lib/src/material/switch.dart#L19
```